### PR TITLE
Add Account Creation Policy for Directories (JSDK-175)

### PR DESCRIPTION
**Pull request by maranai**

### CODE REVIEW

**Comment by maranai** [comment]
Feature related to this PR is already in production. This can be merged immediately when approved.

File: source/java/product-guide/index.markdown [hunk 1]
```diff
@@ -2243,6 +2245,90 @@ In addition to the [search query parameters](#search), you may also use [paginat
...
+
+By default, all of the `Account Creation Policy` email status are set to `DISABLED`.  To modify the policy, just set the desired status to the property that you want to modify:
+
+To enable the ability to send verification or welcome emails for newly created accounts, just set it to `Enabled`.
+
+**Example Request**
+
+    accountCreationPolicy.setWelcomeEmailStatus(EmailStatus.ENABLED)
+        .setVerificationEmailStatus(EmailStatus.ENABLED)
+        .setVerificationSuccessEmailStatus();
```
Comment (mrioan) [inline source/java/product-guide/index.markdown]: missing EmailStatus.ENABLED

File: source/java/product-guide/index.markdown [hunk 2]
```diff
@@ -2243,6 +2245,90 @@ In addition to the [search query parameters](#search), you may also use [paginat
...
+    accountCreationPolicy.setWelcomeEmailStatus(EmailStatus.ENABLED)
+        .setVerificationEmailStatus(EmailStatus.ENABLED)
+        .setVerificationSuccessEmailStatus();        
+    accountCreationPolicy.save();
+
+<a class="anchor" name="modify-account-creation-email-templates"></a>
+#### Modify the Email Templates Used for Verification and Welcome Emails
+
+To modify the emails that get sent during the email verification and registration, let's take a look at the email templates for the account creation policy.  Email templates in Stormpath have common properties that can be modified to change the appearance of the emails.  The properties below apply to both email templates that reside in the account creation policy (verificationEmailTemplates, verificationSuccessEmailTemplates, and welcomeEmailTemplates).
+
```
Comment (mrioan) [inline source/java/product-guide/index.markdown]: This phrase says "both" but the templates enumerated later are 3

File: source/java/product-guide/index.markdown [hunk 3]
```diff
@@ -2243,6 +2245,90 @@ In addition to the [search query parameters](#search), you may also use [paginat
...
+**Resource Attribute for Email Templates**
+
+Attribute | Description | Type | Valid Value
+:----- | :----- | :---- | :----
+`fromEmailAddress` | The address that appears in the email's from field. | String | A valid email address
+`fromName` | The name that appears in the email's from field | String | A string
+`subject` | The subject that appears in the email's subject field | String | A string
+`htmlBody` | The body of the email in HTML format.  This body is only sent when the `mimeType` for the template is set to `text/html`.  This body can take valid HTML snippets. | String | A string. For the resetEmailTemplate it is required to include the macro for the ${url}, ${sptoken} or, ${sptokenNameValuePair}
+`textBody` | The body of the email is plain text format.  This body is only sent when the `mimeType` for the template is set to `text/plain` | String | A string.  For the resetEmailTemplate it is required to include the macro for the ${url}, ${sptoken} or, ${sptokenNameValuePair}
```
Comment (mrioan) [inline source/java/product-guide/index.markdown]: The body of the email IN plain text...

**Comment by mrioan** [comment]
@maranai Review is done. Once the proposed changes are done you can consider this PR approved and merge. Thanks!

**Comment by maranai** [comment]
Thanks @mrioan, I've updated the product guide with your recommendations.

**Pull request merged**